go.mod

```diff
@@ -3,28 +3,26 @@ module github.com/gojek/darkroom
 go 1.12
```

Comment:
The changes look good as some libraries updated to the stable version. Although it is still using go version 1.12, you might want to upgrade to go version 1.15.11 at least. https://endoflife.date/go

Some of these updated libraries might not support an older version. For example, cobra https://github.com/spf13/cobra/blob/master/CHANGELOG.md in the latest changelog still backporting support to go 1.14 (it means they delete the support from older golang version from time to time).